static/docs/commands-reference/status.md

```diff
@@ -120,14 +120,14 @@ cache. For the typical process to update workspaces, see
 ```dvc
 $ dvc status
 
-  bar.dvc
-          outs
-                  changed:  bar
-          deps
-                  changed:  foo
-  foo.dvc
-          outs
-                  changed:  foo
+bar.dvc:
+        changed deps:
```

Comment:
@shcheklein, Check this I have already split the _changed_ into _deps_ and _outs_